### PR TITLE
Port to 1.13

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: cmd/crd-schema-gen/generator/generator.go [hunk 1]
```diff
@@ -66,11 +85,16 @@ func Run() error {
 		fmt.Print(string(out))
 		return err
 	}
+	if err := ioutil.WriteFile(filepath.Join(tmpDir, "PROJECT"), []byte(fmt.Sprintf(`
```
Comment (damemi) [inline cmd/crd-schema-gen/generator/generator.go]: If we merge https://github.com/openshift/kubernetes-sigs-controller-tools/pull/6 you shouldn't need this, right?

Comment (damemi) [reply cmd/crd-schema-gen/generator/generator.go]: Rather, you'd specify them when instantiating the generator

Comment (sttts) [reply cmd/crd-schema-gen/generator/generator.go]: yeah

**Comment by damemi** [comment]
Don't think we need this anymore

**Pull request closed without merging**